docs/BreakingChanges/067 In very specific scenarios, objects cannot be dese.md

```diff
@@ -0,0 +1,30 @@
+## 67: In very specific scenarios, objects cannot be deserialized x-appdomain
```

Comment:
{
  "Id": "67",
  "Title": "In very specific scenarios, objects cannot be deserialized x-appdomain",
  "Details": "TODO",
  "Suggestion": "TODO",
  "Link": "https://msdn.microsoft.com/en-us/library/dn458360(v=vs.110).aspx",
  "Markdown": null,
  "ApplicableApis": [
    "Not detectable via API analysis"
  ],
  "Related": null,
  "VersionBroken": "4.5.1",
  "VersionFixed": null,
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": "Probably too corner-case to be worth trying to spot all the necessary conditions. Description of circumstances and mitigation: https://msdn.microsoft.com/en-us/library/dn458353(v=vs.110).aspx\nSource analyzer status: Not planned",
  "IsRetargeting": false,
  "ImpactScope": 3
}

**Parsed APIs:**
- [ ] Not detectable via API analysis